### PR TITLE
A little configuration cleanup

### PR DESCRIPTION
**Pull request by happygaijin**
@AaronMcKinney

### CODE REVIEW

File: projects/BossCheatSheets.Web.Tests/BossCheatSheets.Web.Tests.csproj [hunk 1]
```diff
@@ -118,7 +118,7 @@
       <Name>BossCheatSheets.DomainModel</Name>
     </ProjectReference>
     <ProjectReference Include="..\BossCheatSheets.Web\BossCheatSheets.Web.csproj">
-      <Project>{BDAB25BF-022A-407B-977A-684E85118C2B}</Project>
+      <Project>{bdab25bf-022a-407b-977a-684e85118c2b}</Project>
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web.Tests/BossCheatSheets.Web.Tests.csproj]: Not sure why it decided to lowercase the guid. meh.

File: projects/BossCheatSheets.Web/BossCheatSheets.Web.csproj [hunk 2]
```diff
@@ -348,7 +349,9 @@
           <AutoAssignPort>True</AutoAssignPort>
           <DevelopmentServerPort>57977</DevelopmentServerPort>
           <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>http://BossCheatSheets.Web</IISUrl>
+          <IISUrl>http://bosscheatsheets.web</IISUrl>
+          <OverrideIISAppRootUrl>True</OverrideIISAppRootUrl>
+          <IISAppRootUrl>http://bosscheatsheets.web</IISAppRootUrl>
           <NTLMAuthentication>False</NTLMAuthentication>
           <UseCustomServer>False</UseCustomServer>
           <CustomServerUrl>
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/BossCheatSheets.Web.csproj]: These are just some little config changes on the project file that I've noticed in the past to help things run a little smoother. no biggie.

File: projects/BossCheatSheets.Web/Web.config [hunk 3]
```diff
@@ -11,7 +11,7 @@
   </configSections>
   <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" throwExceptions="true">
     <targets>
-      <target name="logfile" xsi:type="File" fileName="A:\src\BossCheatSheets\projects\logfile.txt" />
+      <target name="logfile" xsi:type="File" fileName="${basedir}\logs\bosscheatsheets.web.log" />
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/Web.config]: This makes it so it's not hard coded to Aaron's computer :)

File: projects/BossCheatSheets.Web/Web.config [hunk 4]
```diff
@@ -33,7 +33,7 @@
   <system.web>
     <httpRuntime targetFramework="4.5" />
     <compilation debug="true" targetFramework="4.5" />
-    <customErrors mode="On" redirectMode="ResponseRewrite">    
+    <customErrors mode="Off" redirectMode="ResponseRewrite">
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/Web.config]: In dev, we don't need this on. We can use some sort of config transforms when we go to testing and prod later to change this.

File: projects/BossCheatSheets.Web/packages.config [hunk 5]
```diff
@@ -38,6 +38,5 @@
   <package id="structuremap" version="3.0.4.125" targetFramework="net45" />
   <package id="structuremap.web" version="3.0.0.108" targetFramework="net45" />
   <package id="Twitter.Bootstrap" version="3.0.1.1" targetFramework="net45" />
-  <package id="WebGrease" version="1.3.0" targetFramework="net45" />
   <package id="WebGrease" version="1.5.2" targetFramework="net45" />
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/packages.config]: This was causing the project to fail. Not sure why/how there are two versions. But I removed the older one.

File: projects/BossCheatSheets.Web/Web.config (hunk 3 above)
Comment (guavajellyaaron) [inline projects/BossCheatSheets.Web/Web.config]: I was wondering about this... then forgot about it...

**Comment by guavajellyaaron** [comment]
Looks good. Thanks for the cleanup help.

**Pull request merged**